lib/lightning_web/live/components/common.ex

```diff
@@ -174,7 +174,7 @@ defmodule LightningWeb.Components.Common do
     assigns = assign(assigns, has_tooltip?: has_tooltip?, styles: styles)
 
     ~H"""
-    <div id={"#{@id}-container"} class="flex items-baseline text-sm font-normal">
+    <div id={"#{@id}-container"} class="flex items-middle text-sm font-normal">
```

Comment:
Hey @elias-ba , I can't seem to find `item-middle` in the tailwind v3 docs.

Reply:
Ah ! it should be items-center. But this proves that the css class is useless their. Thanks for the catch, I will remove it

Reply:
Niice